docs/talkback-guidelines.md

```diff
@@ -0,0 +1,347 @@
+
+* The foreground color of the view in question was located and it was determined that it was `<color name="wp_grey_dark">#2e4453</color>`
+* Luckily for us, we have several other colors that are higher in contrast that can be utilized. 
+
+	```xml
+	<color name="wp_grey_dark">#2e4453</color>
+	<color name="wp_grey_darken_10">#668eaa</color>
+	<color name="wp_grey_darken_20">#4f748e</color>
+	<color name="wp_grey_darken_30">#3d596d</color>
+	```
+* The `wp_grey_darken_10` was then checked with the Contrast Checker with the background color of white and the ratio was above the amount stipulated for low vision users. 
+
+This was the end result : 
+
+Before | After 
+--------|-------
+<img src="https://user-images.githubusercontent.com/1509205/71564967-16c97e00-2a77-11ea-8372-04791708408a.png" width="320">        | <img src="https://user-images.githubusercontent.com/1509205/71564969-1fba4f80-2a77-11ea-84e7-3353ba5c7454.png" width="320">
+
+Even though the change is subtle, improving the color contrasts of our elements can go a far way for users with low vision. 
+ 
+### <a name="grouping-content"></a>Grouping content
+If users should treat a set of elements as a single unit of information, you can group these elements in a focusable container using `android:focusable=”true”` or `android:screenReaderFocusable` attribute to `true` in API 28 and above.  
+
+<img src="images/accessibility-guidelines/stats_group.png" width="300">
+
+In cases like these, it's important that the descriptions of the elements are kept short since they all will be announced in a single utterance. 
+
+### <a name="live-regions"></a>Live regions (dynamic content)
+
+To activate the functionality of views with TalkBack you have to navigate to it. However there are times when the contents of a view might changeand the user should be made aware of it. This is where Live Regions come in. For example say you have a `TextView` similar to the one defined below. 
+
+```xml
+<TextView
+   android:id="@+id/feedback_text_view"
+   android:layout_width="match_parent"
+   android:layout_height="wrap_content"
+   android:textColor="@color/white"
+   android:padding="@dimen/standard_padding"
+   android:textSize="@dimen/large_text" />
+```
+
+You can add a live region attribute to it so that when `setText` is called TalkBack will announce it. 
+
+```xml
+<TextView
+   android:id="@+id/feedback_text_view"
+   android:layout_width="match_parent"
+   android:layout_height="wrap_content"
+   android:textColor="@color/white"
+   android:padding="@dimen/standard_padding"
+   android:textSize="@dimen/large_text" 
+   android:accessibilityLiveRegion="polite"/>
+```
+
+The live region attribute is set to `polite` so that means it won't interrupt TalkBack if it is currently announcing something but if it is important that it gets announced it can be set to `assertive`. This should be used sparing.
```

Comment:
```suggestion
The live region attribute is set to `polite` so that means it won't interrupt TalkBack if it is currently announcing something but if it is important that it gets announced it can be set to `assertive`. This should be used sparingly.
```

I think you meant “sparingly” here. 🙂

Reply:
I did. Good catch 😄